shadowsocks2/Dockerfile

```diff
@@ -20,4 +20,8 @@ RUN apk upgrade --update \
 
 COPY --from=builder /go/bin/go-shadowsocks2 /usr/bin/shadowsocks
 
-ENTRYPOINT ["shadowsocks"]
+ADD entrypoint.sh /entrypoint.sh
+
+RUN chmod +x /entrypoint.sh
```

Comment:
@crabhit I think adding executable permissions can be done locally because git can save it.

Reply:
Make sense. Changed executable permission and removed this line.